requirements-test.txt

```diff
@@ -29,3 +29,6 @@ fairscale >= 0.4.5
 # Dependency for fused layers, optional
 triton==2.0.0.dev20221105
 networkx
+
+# Dependency for triton flash attn
+flash-attn
```

Comment:
Looks like we would need to change our build scripts to also install `flash-attn`. Otherwise we might have this override our version of `_C_flashattn` that we compile

Reply:
I think they're skipping because I'm requiring sm80 for the Triton implementation

Reply:
I added flash-attention as a dependency in setup.py, is that what you had in mind?

Reply:
> I think they're skipping because I'm requiring sm80 for the Triton implementation

Indeed we don't have sm80 on the CI. But it should work with Sm75 (we have it in the CI)